data-preservation-programs/spade-tenant
# issue-45

---

<details open="true"><summary>Generated summary (powered by <a href="https://app.graphite.dev">Graphite</a>)</summary>

> # Pull Request Description
> 
> ## TL;DR
> This pull request implements the logic to run the sp filtering job. The job will run periodically to figure out which sp are eligible for a tenant based on the eligibility criteria specified by the tenant. .
> 
> ## What changed
> - Schema changed from text to jsonb for clause_value
> - Reduced scope of comparison operators 
> - Code to filter sp for a tenant 
> 
> ## How to test
> 1. Compile and run the code.
> 2. Verify that the code compiles successfully.
> 3. Test the functionality of the comparison operators.
> 4. Verify that the code behaves as expected.
> 
> ---
